src/modules/shared/layouts/AppLayout/AppLayout.tsx

```diff
@@ -332,7 +332,7 @@ const AppLayout: FC<any> = ({ children }) => {
 				node: (
 					<div
 						onClick={() => {
-							window.open(window.location.origin, '_self');
+							window.open(`${window.location.origin}/${locale}`, '_self');
```

Comment:
We could use a link for this, so we don't have to reload the complete react app:
It also shows the link you will navigate to at the bottom of the screen when we use a link:
![image](https://github.com/user-attachments/assets/877336aa-2d0f-48b2-a840-9f8900243e2a)

```typescript
				node: (
					<Link href={'/' + locale} passHref>
						<a className="c-navigation__logo--hamburger">
							<HetArchiefLogo
								className="c-navigation__logo c-navigation__logo--list"
								type={isMobile ? HetArchiefLogoType.Dark : HetArchiefLogoType.Light}
							/>
						</a>
					</Link>
				),
```